### PR TITLE
bump avago + evm at e2e

### PR DESCRIPTION
**Pull request by felipemadero**

### CODE REVIEW

File: scripts/tests.e2e.sh [hunk 1]
```diff
@@ -12,9 +12,9 @@ fi
 
 ANR_PATH=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
 
-DEFAULT_VERSION_1=v1.10.10
-DEFAULT_VERSION_2=v1.10.9
-DEFAULT_SUBNET_EVM_VERSION=v0.5.5
+DEFAULT_VERSION_1=v1.10.15
+DEFAULT_VERSION_2=v1.10.14
+DEFAULT_SUBNET_EVM_VERSION=v0.5.9
```
Comment (ceyonur) [inline scripts/tests.e2e.sh:17]: subnet-evm 0.5.9 won't work with avago v1.10.14. IMO If these different versions need to be run with subnet-evm; we should define two different versions for subnet-evm as well.

Comment (ceyonur) [reply scripts/tests.e2e.sh:17]: https://github.com/ava-labs/subnet-evm#avalanchego-compatibility

Comment (felipemadero) [reply scripts/tests.e2e.sh:17]: DEFAULT_VERSION_ is not used in combination with DEFAULT_SUBNET_EVM_VERSION, it is just used for a node restart test without evm involved

Comment (felipemadero) [reply scripts/tests.e2e.sh:17]: compatibility must exist between DEFAULT_VERSION_1 and DEFAULT_SUBNET_EVM_VERSION only
would you like a change in names, or some comment, to clarify that?

Comment (ceyonur) [reply scripts/tests.e2e.sh:17]: comment would be nice

**Review by ceyonur — APPROVED** [review]

**Pull request merged**